Chavah.NetCore/Common/HealthChecks/HealthCheckBuilderExtensions.cs

```diff
@@ -15,22 +15,20 @@ namespace Microsoft.Extensions.DependencyInjection
     public static class HealthCheckBuilderExtensions
     {
         /// <summary>
-        /// Add a healthcheck for ravendb.
+        /// Add a HealthCheck for RavenDb.
         /// </summary>
-        /// <param name="builder"></param>
-        /// <param name="services"></param>
-        /// <param name="name"></param>
-        /// <param name="failureStatus"></param>
-        /// <param name="tags"></param>
+        /// <param name="builder">The <see cref="IHealthChecksBuilder"/>.</param>
+        /// <param name="name">The name of the HealthCheck.</param>
+        /// <param name="failureStatus">The <see cref="HealthStatus"/>The type should be reported when the health check fails. Optional. If <see langword="null"/> then</param>
+        /// <param name="tags">A list of tags that can be used to filter sets of health checks. Optional.</param>
         /// <returns></returns>
         public static IHealthChecksBuilder AddRavenDbCheck(
             this IHealthChecksBuilder builder,
-            IServiceCollection services,
             string name = "ravendb",
             HealthStatus? failureStatus = default,
             IEnumerable<string> tags = default)
         {
-            var provider = services.BuildServiceProvider();
+            var provider = builder.Services.BuildServiceProvider();
```

Comment:
I understand calling BuildServiceProvider is usually a red flag: https://twitter.com/davidfowl/status/1038521947007594498

Is it required here?

Reply:
This will be called as the App is being built, so I am not sure what the other use cases are. I am using it in various places. Let me know if I need to modify it.

Reply:
Good catch, I refactored this. Please see new PR.